guest-dart-sdk/docs/Image.md

```diff
@@ -0,0 +1,15 @@
+# guest_sdk.model.Image
```

Comment:
There's no description to describe what's the plan with this PR, so I don't know why you'd manually change the dart SDK at all.

What's happening here?
Are you manually creating a new SDK? Seems like that should be a whole different PR